assemblers/io.weldr.qcow2

```diff
@@ -45,7 +45,7 @@ def loop_device(image, size, offset=0):
 
 def main(tree, output_dir, options):
     filename = options["filename"]
-    partition_table_id = options["partition_table_id"]
+    # partition_table_id = options["partition_table_id"]
```

Comment:
Was this by accident?

Reply:
The argument is unused and should be dropped entirely (could be in a follow-up though).